### DISCUSSION TITLE
2024/10/07/%E5%9B%BD%E5%BA%86%E5%B0%8F%E6%80%9D/

### DISCUSSION THREAD

**Discussion by giscus** [Jared-ZDC/markel#41] [General]
# 2024/10/07/%E5%9B%BD%E5%BA%86%E5%B0%8F%E6%80%9D/

国庆期间跟朋友聊天，关于职业天花板的问题，这个问题也是我这段时间一直在思考或者说困惑的问题； 坦白说，目前做到当前的程度，个人其实是比较满意的，这期间花了很多很多的时间精力在工作上，在生活上面，泛善可陈，当然这里也有自己的原因，没有好好的去经营，但是不得不说，工作对于精力的消耗是巨大的，往往在有时间的时候，已经没有精力去更好的享受生活； 我一直认为，人没有上辈子，也没有下辈子，只有这辈子！一旦逝去

https://markel.top/2024/10/07/%E5%9B%BD%E5%BA%86%E5%B0%8F%E6%80%9D/?t=1728298234706

**Comment by Jared-ZDC** [comment]
当前进度 11.3%
